components/avatar/demo/basic.md

```diff
@@ -0,0 +1,45 @@
+---
+order: 0
+title:
+  zh-CN: 基本
+  en-US: Basic
+---
+
+## zh-CN
+
+头像有三种尺寸，两种形状可选。
+
+## en-US
+
+Three sizes and two shapes are available.
+
+```` html
+<template>
+  <div id="components-avatar-demo-basic">
```

Comment:
no need id and extra style. if needed, use inline-style like antd

Reply:
you are right. I can use inline-style at present, but antd user '<style>'.Maybe it will be changed later.